java/arcs/core/util/UnionFind.kt

```diff
@@ -34,23 +46,33 @@ class UnionFind<E> {
      */
     fun find(e: E): E = e.findRoot().element
 
+    /**
+     * Update the information for the equivalence class for [e] with [info].
+     */
+    fun setInfo(e: E, info: I) { e.findRoot().info = info }
```

Comment:
If a function has a body that needs to be wrapped in braces, the body should go on its own line.

```kotlin
fun setInfo(e: E, info: I) {
    e.findRoot().info = info
}
```

Reply:
Ah, thought I was being clever. ;) Fixed it.